src/js/disability-benefits/components/ClaimEstimationPage.jsx

```diff
@@ -16,7 +17,7 @@ export default class ClaimEstimationPage extends React.Component {
             <nav className="va-nav-breadcrumbs">
               <ul className="row va-nav-breadcrumbs-list" role="menubar" aria-label="Primary">
                 <li><Link to="your-claims">Your claims</Link></li>
-                <li><Link to={`your-claims/${this.props.params.id}/status`}>Your Disability Compensation Claim</Link></li>
+                <li><Link to={`your-claims/${this.props.params.id}/status`}>Your {getClaimType(this.props)} Claim</Link></li>
```

Comment:
That looks right to me. We should move this file to the `containers` directory, though, since our convention is that page components that use Redux go there and not in `components/`.